page-objects/leftMenu.ts

```diff
@@ -1,17 +1,20 @@
 import { Locator, Page, expect } from "@playwright/test";
-import { logClicking, logTyping } from "../utils/decorators";
+import { logClicking, logHoveringOverElement, logTyping } from "../utils/decorators";
 
 export class LeftMenu {
   private readonly page: Page;
   private readonly leftSideBar: Locator;
   private readonly renameInput: Locator;
+  private readonly folderNameInput: Locator;
 
   private menuElement: Locator;
+  private spaceElement: Locator;
 
   constructor(page: Page) {
     this.page = page;
     this.leftSideBar = this.page.locator("cu-simple-bar");
     this.renameInput = this.leftSideBar.getByTestId("nav-editor__input");
+    this.folderNameInput = this.page.getByTestId('nav-editor__input');
```

Comment:
Lokator `folderNameInput` jest taki sam, jak `renameInput` powyżej; generalnie chyba dla każdego elementu w menu po lewej, jeżeli zmienia się jego nazwę, to ten input ma taki sam lokator

Reply:
o faktycznie nie wiem czy to mi nie doszło w jakimś mergu